contracts/bridge/contracts/binance/LightClientBridge.sol

```diff
@@ -611,6 +647,11 @@ contract LightClientBridge is Bitfield, ValidatorRegistry, GuardRegistry {
         view
         returns (uint256)
     {
+        require(
+            block.number < blockNumber + 256,
```

Comment:
Could we extend the scope to following here?

`block.number <= blockNumber + 256 + BLOCK_WAIT_PERIOD`

Need to check the corner case of equal will works or not, I'm not sure wether we can fetch the blockhash when equal?

Reply:
And will not work if

`block.number < blockNumber + BLOCK_WAIT_PERIOD`.

Not sure use < or <= here too.

Reply:
> And will not work if
> 
> `block.number < blockNumber + BLOCK_WAIT_PERIOD`.
> 
> Not sure use < or <= here too.

Ah, this is checked in caller functions: `verifyCommitment` and `createRandomBitfield`.

Then, can we reuse `createRandomBitfield` here?

Reply:
1. `blockhah` works for 256 most recent in Ropsten.
2. `block.number < blockNumber + BLOCK_WAIT_PERIOD`, In this case, It should not get the seed.
```
        require(
            block.number >= data.blockNumber + BLOCK_WAIT_PERIOD,
            "Bridge: Block wait period not over"
        );
```

Reply:
Not `block.number < blockNumber + BLOCK_WAIT_PERIOD`, but `block.number <= blockNumber + 256 + BLOCK_WAIT_PERIOD`